addons/account/models/account_move.py

```diff
@@ -5079,6 +5079,16 @@ def is_internal_partner(partner):
         if msg_dict.get('subject') and msg_dict.get('body'):
             msg_dict['body'] = Markup('<div><div><h3>%s</h3></div>%s</div>') % (msg_dict['subject'], msg_dict['body'])
 
+        # Don't create the move if no attachment.
+        if len(msg_dict['attachments']) < 1:
+            mail_template = self.env.ref('account.email_template_mail_gateway_failed')
+            lang = (partners and partners[0].lang) or self.env.user.lang
+            mail_template.with_context(lang=lang).send_mail(self.env.company.id, force_send=True, email_values={
+                'email_to': from_mail_addresses[0],
+                'email_from': msg_dict['to'],
```

Comment:
If you don't set a reply_to it might just email_from to respond, so auto-response bots would cause a loop.
Would probably just not set email_from as email_to here, doesn't really make sense. Can just use the default sender email, or the company bounce email as is done in the bouncing code.